integrations/pom.xml

```diff
@@ -12,9 +12,9 @@
   <name>Configuration as Code Integration tests</name>
 
   <properties>
-    <jenkins.version>2.176.3</jenkins.version> <!-- Remove when parent base is updated. -->
     <maven.deploy.skip>true</maven.deploy.skip> <!-- no need to be deployed during release, this is a test-only module -->
     <jackson.version>2.10.2</jackson.version>
+    <surefire.rerunFailingTestsCount>4</surefire.rerunFailingTestsCount>
```

Comment:
This is a risky change. It usually hides flaky tests, or even worse, production race conditions.

Reply:
FWIW I wouldn't merge with this change

Reply:
Leaving this property was never my intention. My proposal was:
> * Set surefire.rerunFailingTestsCount to 4 (as we had in the past parent pom) for integrations/pom.xml
> * Launch several times the CI to decide if we should skip the test for java 11. Taking into consideration that the test passes on Windows and Linux for the github checks, I don't actually think it's rising a real problem

I might not explain my proposal properly, but my intention was to try to see the flakiness of the test and if agreed, skip it on java 11 since it only fails sometimes with jenkins.io CI executing with java 11.